app/assets/javascripts/index/directions/graphhopper.js

```diff
@@ -79,5 +80,6 @@ function GraphHopperEngine(id, vehicleType) {
   };
 }
 
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_bicycle", "bike"), true);
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_foot", "foot"), true);
+OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_car", "car", true), true);
```

Comment:
This is true but we have to enable CH for backward compatibility reasons for now